src/components/RandomDog.js

```diff
@@ -1,15 +1,29 @@
 import React from "react";
 import "./RandomDog.css";
 
-export default function RandomDog() {
+function RandomDog(props) {
+  
+  // {
+  //     "message": "https://images.dog.ceo/breeds/hound-ibizan/n02091244_4281.jpg",
+  //     "status": "success"
+  // }
+
+console.log("props",props)
+
```

Comment:
delete console.log always after you are sure your code is working and you want to push final code